src/main/java/com/google/devtools/build/lib/sandbox/SandboxModule.java

```diff
@@ -358,22 +358,28 @@ private void setup(CommandEnvironment cmdEnv, SpawnStrategyRegistry.Builder buil
 
   @Nullable
   private static Path getPathToDockerClient(CommandEnvironment cmdEnv) {
+    SandboxOptions options = cmdEnv.getOptions().getOptions(SandboxOptions.class);
+    FileSystem fs = cmdEnv.getRuntime().getFileSystem();
+
+    if (fs.exists(PathFragment.create(options.dockerSandboxExecutable))) {
```

Comment:
`FileSystem` methods are not meant to be called directly. Instead, use `getPath` and then call the methods on the `Path` instance.

@tjgq I guess that's the drawback of my recent change that made these methods public?